File: app/src/main/java/xyz/klinker/messenger/fragment/settings/MaterialPreferenceFragmentCompat.java
package xyz.klinker.messenger.fragment.settings;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.preference.PreferenceFragmentCompat;
import android.view.View;

import xyz.klinker.messenger.R;

public abstract class MaterialPreferenceFragmentCompat extends PreferenceFragmentCompat {

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        getListView().setBackgroundColor(getResources().getColor(R.color.drawerBackground));
    }
}

File: app/src/main/java/xyz/klinker/messenger/fragment/settings/MaterialPreferenceFragmentCompat.kt
package xyz.klinker.messenger.fragment.settings

import android.os.Bundle
import android.view.View
import androidx.preference.Preference
import androidx.preference.PreferenceFragmentCompat

import xyz.klinker.messenger.R

@Suppress("DEPRECATION")
abstract class MaterialPreferenceFragmentCompat : PreferenceFragmentCompat() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        listView.setBackgroundColor(resources.getColor(R.color.drawerBackground))
    }

    fun findPreference(key: String) = findPreference<Preference>(key as CharSequence)!!
}
